nwbwidgets/controllers/group_and_sort_controllers.py

```diff
@@ -293,13 +307,12 @@ def get_orderable_cols(self):
         return [x for x in candidate_cols if len(robust_unique(self.units[x][:])) > 1]
 
     def group_and_sort(self):
-        if self.group_vals is None and self.order_vals is None:
+        if self.column_values is None and self.order_vals is None:
```

Comment:
why are you changing the name of this variable from group_vals to column_values?